### DISCUSSION TITLE
[Feature request] Make Pluto built-in package manager support github links (if it isn't supported yet)

### DISCUSSION THREAD

**Discussion by Alseidon** [fonsp/Pluto.jl#2732] [Pluto development] [answered]
I love using Pluto, which makes everything flow smoothly, but I have to use a package which isn't in the general registry (yet?), meaning I have to use `Pkg.add(url="https://github.com/user/package.jl")`. Is there a way to reproduce this behaviour without disabling the built-in package manager? If not, could this be "easily" supported?

**Comment by disberd** [comment, marked as answer]
There is an open draft PR by me https://github.com/fonsp/Pluto.jl/pull/2245 which has been stale for a long time.
I still didn't find the time to update it and work on it again, but unfortunately it is currently not possible before this PR (or something else similar is implemented).

**Reply by Alseidon** [reply]
Thank you for your answer!
